src/lxc/caps.c

```diff
@@ -247,6 +244,8 @@ bool lxc_file_cap_is_set(const char *path, cap_value_t cap, cap_flag_t flag)
 	cap_is_set = lxc_cap_is_set(caps, cap, flag);
 	cap_free(caps);
 	return cap_is_set;
+	#else
+	return true;
```

Comment:
But if cap_get_file() call itself fails, you return false :)

i think returning false, and demanding different privilege, is probably best.  Or will that break android?  Is there a platform out there that relies on filecaps but then doesn't offer a libcap that can read them?